packages/SwingSet/src/kernel/vatManager/supervisor-nodeworker.js

```diff
@@ -4,13 +4,14 @@
 import '@agoric/install-ses';
 import { parentPort } from 'worker_threads';
 import anylogger from 'anylogger';
+import engineGC from 'expose-gc/function';
```

Comment:
I'm a little surprised this works, since it means the NodeWorker thread can `require('v8')` and `require('vm')`, and I thought NodeWorkers were slightly less powerful than that.

Reply:
NodeWorkers can `require(ANYTHING)`.